backend/fregepoc/indexers/models.py

```diff
@@ -73,3 +80,50 @@ def __iter__(self):
                 self.rate_limit_exceeded = True
                 break
             yield from repos_to_process
+
+
+class SourceforgeIndexer(SingletonModel, BaseIndexer):
```

Comment:
Could you also register this model in the Django Admin Panel?

Reply:
Perhaps, it will not be needed. There is an open MR: #11  with a solution to do it automatically. However, should it get merged before this one, you will need to remove `SingletonModel` from the inheritance list and make some other minor adjustments.